package.json

```diff
@@ -37,6 +37,8 @@
     "starts": "cross-env NODE_ENV=development webpack-dev-server --https --colors --config ./build_scripts/webpack.config.js",
     "start-prod": "cross-env NODE_ENV=production webpack-dev-server --colors --config ./build_scripts/webpack.config.js",
     "test": "npm run lint && npm run jest",
+    "test-unit": "test",
+    "test-e2e": "npx concurrently --kill-others -n 'neo4j,cypress,browser' 'docker run --env NEO4J_AUTH=neo4j/newpassword -p7687:7687 neo4j:4.2.2' 'yarn wait-on-neo4j && yarn wait-on-dev && yarn e2e-local --env server=4.2' 'yarn && yarn start'",
```

Comment:
This is failing on my machine, wait-on times out. Neo4j and yarn start seem to start as expected and on expected ports, but then nothing happens until wait-on times out.

Reply:
port 7474 should be forwarded

Reply:
Thanks!